### PR TITLE
registering gunishmatta

### PR DESCRIPTION
**Pull request by gunishmatta**
Hey Maintainer!
Please accept my request!

### CODE REVIEW

File: content/participants/ttjc@gunishmatta/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+I love to learn new things.
+
+### help I need?
+
+It would be good if someone tells me how to do build complex projects with React and Redux Saga and a little bit advanced Backend.
+
+### how to contact me?
+
+- Twitter
```
Comment (khusharth) [inline content/participants/ttjc@gunishmatta/index.md:31]: Hey! @gunishmatta you need to add links to these platforms too in markdown format like:
```
[Twitter](link-to-your-twitter-profile-here)
```

File: content/participants/ttjc@gunishmatta/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+type the name of dish we want to search for it fetches data and display all the
+recipes of dishes matching with our query.
+
+_stack_ Made in React with hooks/
+
+_hosted link_ https://gkvr5.csb.app/
+
+_github link_ https://github.com/gunishmatta/Food-Recipe-App
+
+#### Bicoin Tracker
```
Comment (khusharth) [inline content/participants/ttjc@gunishmatta/index.md:52]: Bitcoin*

File: content/participants/ttjc@gunishmatta/index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+_description_ A Realtime Bitcoin Tracker using Socket.io,React,Charts.js and Coinbase API.
+
+_stack_ React,Charts.js,Socket.io
+
+_hosted link_ https://csb-stqql-ek5t0ywfv.now.sh/
```
Comment (khusharth) [inline content/participants/ttjc@gunishmatta/index.md:58]: You also need to add the github link for the project here.

File: content/participants/ttjc@gunishmatta/index.md (hunk 1 above)
Comment (gunishmatta) [reply content/participants/ttjc@gunishmatta/index.md:31]: Thanks a lot ...I'll update it accordingly

**Comment by gunishmatta** [comment]
I have done the necessary changes

**Pull request merged**